.travis.yml

```diff
@@ -10,15 +10,11 @@ python:
   - 3.7
   - 3.8
 
-before_install:
-  - pip install -r requirements-dev.txt
```

Comment:
We need this for other stuff that will be added later. Can you revert this?

Reply:
Umm, iirc if they are installed the list at https://github.com/jackmaney/python-stdlib-list/pull/30/files#diff-9e9ddedef65dec3da86063a067839e65R36 becomes quite large.

I suggest when they are needed on Travis, they are installed inside a tox testenv.  I didnt delete the file - it is still available to be used for other Travis jobs, e.g. a docs or publish job, and can be used by devs locally.

Reply:
Makes sense. Let's keep them away for now and I'll send another PR later for this.

Let's keep this in! Thanks for the tests and improvements. I'll see if I can get a new release out after this.